locales/fr.schema.json

```diff
@@ -388,7 +388,7 @@
           "label": "Petit"
         },
         "options__2": {
-          "label": "Moyen"
+          "label": "Média"
```

Comment:
This isn't a good translation of `medium`. Here in this context we're talking about sizes so it should stay `Moyen`